client/src/components/base/TXLoading.tsx

```diff
@@ -0,0 +1,38 @@
+import { Link } from "react-router-dom";
+import { useDispatch } from "react-redux";
+import { grantsPath } from "../../routes";
+import { resetFileStatus } from "../../actions/ipfs";
+import { resetTXStatus } from "../../actions/newGrant";
+
+function TXLoading({ status }: { status: string }) {
+  const dispatch = useDispatch();
+  const resetStatuses = () => {
+    dispatch(resetFileStatus());
+    dispatch(resetTXStatus());
+  };
+  if (status === "initiated") {
```

Comment:
I would use a constant so that if we change/remove it it would raise an error at compile time